File: src/main/java/coffee/amo/interactionkey/api/event/InteractionKeyPressedEvent.java
package coffee.amo.interactionkey.api.event;

import net.minecraft.world.entity.player.Player;
import net.minecraftforge.event.entity.player.PlayerEvent;
import net.minecraftforge.eventbus.api.Event;

public class InteractionKeyPressedEvent extends PlayerEvent {

    public InteractionKeyPressedEvent(Player player) {
        super(player);
    }
}

File: src/main/kotlin/coffee/amo/interactionkey/api/event/InteractionKeyPressedEvent.kt
package coffee.amo.interactionkey.api.event

import net.minecraft.core.BlockPos
import net.minecraft.world.entity.player.Player
import net.minecraftforge.event.entity.player.PlayerEvent

class InteractionKeyPressedEvent(val player: Player?, val blockPos: BlockPos?, val entityID: Int?) : PlayerEvent(player)